ci/test_suites/authenticators_k8s/dev/Dockerfile.test

```diff
@@ -1,13 +1,8 @@
-FROM google/cloud-sdk:388.0.0
+FROM google/cloud-sdk:424.0.0
 
 RUN mkdir -p /src
 WORKDIR /src
 
-# Install gcloud authentication plugin
-RUN apt-get update && \
-    apt-get install -y google-cloud-sdk-gke-gcloud-auth-plugin && \
-    apt-get clean
-
```

Comment:
Hey @telday, what's the basis for removing this? I don't see anything in the PR description or commit message for why this can now be removed. Thanks!